Dockerfile

```diff
@@ -1,5 +1,27 @@
 FROM redash/base:latest
 
+ENV DEBIAN_FRONTEND=noninteractive \
+    MOZ_HEADLESS=1 \
+    FIREFOX_VERSION=59.0 \
+    GECKODRIVER_VERSION=0.20.0
+
+# Install requirements for UI tests
+RUN dependencies=' \
```

Comment:
Any reason to do multi-line strings here instead of passing it to apt-get directly?

Reply:
I tend to use multi-line strings for dependency installs because it makes reviewing pull requests that add or remove dependencies easier. In this particular case, with only 3 dependencies, it doesn't make a huge difference.